continuous_integration/travis/test_script.sh

```diff
@@ -22,11 +22,11 @@ fi
 if [[ "$SKLEARN_TESTS" == "true" ]]; then
     # Install scikit-learn from conda and test against the installed
     # development version of joblib.
-    conda install --yes cython pillow scikit-learn scipy==1.2.1
+    python -mpip install cython==0.29 pillow==6.0 scipy==1.2.1 scikit-learn==0.20.3
```

Comment:
Let's test for the latest stable version of everything:

```suggestion
    python -m pip install cython pillow scipy scikit-learn
```